lib/config/presets/internal/default.ts

```diff
@@ -538,7 +539,7 @@ export const presets: Record<string, Preset> = {
     ],
   },
   dependencyDashboard: {
-    description: 'Enable Renovate Dependency Dashboard creation.',
+    description: gt.gettext('Enable Renovate Dependency Dashboard creation.'),
```

Comment:
It can be tricky here, preset descriptions well be loaded before and configuration loaded. So `gt.text(...)` here is mere to let `exgettext` program extracting `msgid`, it will aways be "translated" to English here, and then those text should be re-evaluated in some where it actually shown up.

Reply:
There is another way to implement this: Let description to be an anonymous function like:
```typescript
(gt: Gettext) => {
    gt.gettext('...');
}
```
then invoke that function at some where it should be shown up, and passed initialized `gt` in.